src/compiler/printer/wast/index.js

```diff
@@ -0,0 +1,442 @@
+// @flow
+/* eslint flowtype-errors/show-errors: warn */
```

Comment:
I had to disable flow unfortunately, something is not working with the AST nodes definition.